db/monitor/DELA133.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<monitor name="Dell U3219Q" init="standard">
+  <caps add="(prot(monitor)type(LCD)model(U3219Q)cmds(01 02 03 07 0C E3 F3)vcp(02 04 05 08 10 12 14(05 08 0B 0C) 16 18 1A 52 60(01 0F 11 ) AA(01 02 04 ) AC AE B2 B6 C6 C8 C9 CC(02 0A 03 04 08 09 0D 06 ) D6(01 04 05) DC(00 03 05 ) DF E0 E1 E2(00 1D 02 04 0E 12 14 ) F0(0C ) F1 F2 FD)mswhql(1)asset_eep(40)mccs_ver(2.1))/">
```

Comment:
I've confirmed that the model number and controls are correct, but I'm not sure how to make sure this line is correct. I've set the correct model number within this line, but I suspect the rest is wrong. When I insert this into the ddccontrol-db on my computer, I get "document of the wrong type, can't find controls or include". Since the file exists with these properties, I suspect that they're not found because caps don't match what was discovered. When I run `sudo ddccontrol -p -c -d`, I get:

```
Capabilities:
ioctl(): Inappropriate ioctl for device
ioctl returned -1
ioctl(): Inappropriate ioctl for device
ioctl returned -1
ioctl(): Inappropriate ioctl for device
ioctl returned -1
ioctl(): Inappropriate ioctl for device
ioctl returned -1
ioctl(): Inappropriate ioctl for device
ioctl returned -1
ioctl(): Inappropriate ioctl for device
ioctl returned -1
ioctl(): Inappropriate ioctl for device
ioctl returned -1
ioctl(): Inappropriate ioctl for device
ioctl returned -1
ioctl(): Inappropriate ioctl for device
ioctl returned -1
Capabilities read fail.
```

not sure how to handle that - would love pointers!

Reply:
But, when I remove that <caps> element and let <controls> and <include> exist under <monitor>, the definition loads correctly.

Reply:
@hughrawlinson Hi! Did you come to a conclusion about this? Should I merge this file?

Reply:
I don't really know, I don't know how to get the correct line for that - it's been working fine for me so it's probably ok.

Reply:
Okay, I don't have any way to confirm it, so I'll just take your word for it. :-)